firebaseConfig.ts

```diff
@@ -0,0 +1,27 @@
+import admin  from 'firebase-admin'
+import {envDetails} from './utils/environmentDetails'
+
+if (admin.apps.length > 0) {
```

Comment:
If I understand correctly, this is production mode? 
It would be good to move this into a variable that makes it clear. 

```suggestion
const isProdMode = admin.apps.length > 0
if (isProdMode) {
```